instagram/src/App.js

```diff
@@ -1,24 +1,71 @@
 import React, { Component } from 'react';
 import './App.css';
-import dummyData from "./dummy-data";
-import SearchBar from "./Components/SearchBar/SearchBar";
-import PostContainer from "./Components/PostContainer/PostContainer";
+import SearchBar from './Components/SearchBar/SearchBar';
+import PostContainer from './Components/PostContainer/PostContainer';
+import postData from './dummy-data.js';
 
 class App extends Component {
-  constructor() {
+
+  constructor(props) {
     super();
+    
+    this.state = {
+      allPosts: [],
+      displayedPosts: []
+    }
+
+    this.filterPosts = this.filterPosts.bind(this);
+  }
+
+ 
+
+  componentDidMount() {
+    this.setState({
+      allPosts: postData,
+      displayedPosts: postData
+    });
```

Comment:
Good job bringing in componentDidMount. This is the perfect place to load your data to the state, this is also a good place to fetch data from server.